changelogs/fragments/1596-xfs_quota-feedback_on_projects_not_initialized_has_changed.yml

```diff
@@ -0,0 +1,3 @@
+---
+bugfixes:
+  - xfs_quota - The feedback for initializing project quota using xfs_quota binary from `xfsprogs` has changed since the version it was written for.
```

Comment:
```suggestion
  - xfs_quota - the feedback for initializing project quota using xfs_quota binary from ``xfsprogs`` has changed since the version it was written for (https://github.com/ansible-collections/community.general/pull/1596).
```